.github/ISSUE_TEMPLATE/troubleshooting-question.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Troubleshooting question
+about: Ask a question to solve installation and configuration issues
+
+---
+
+For troubleshooting, see [https://www.mattermost.org/troubleshoot/](https://www.mattermost.org/troubleshoot/).
```

Comment:
Troubleshooting for plugins mostly happens in GitHub issues. To which repos does the troubleshooting steps apply?